service/chartconfig/v1/helm/error.go

```diff
@@ -26,9 +27,16 @@ func IsReleaseNotFound(err error) bool {
 	if strings.HasPrefix(c.Error(), releaseNotFoundErrorPrefix) {
 		return true
 	}
-	if c == releaseNotFoundError {
+	if strings.HasSuffix(c.Error(), releaseNotFoundErrorSuffix) {
```

Comment:
Lets please keep the exact type match as well.

Reply:
Sorry, that got lost. I've added it back.